app/views/candidate_mailer/application_under_consideration.text.erb

```diff
@@ -0,0 +1,15 @@
+Dear <%= @candidate_name %>,
+
+# Your application is being considered
+
+Your application is now with your teacher training provider(s).
```

Comment:
We can use `pluralize` to get rid of the `(s)`:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/blob/fe5d289ce603ca5ffea3b19a61acf05e4b51e672/app/views/candidate_mailer/submit_application_email.text.erb#L23

Reply:
:+1: :+1: